nuitka/plugins/PluginBase.py

```diff
@@ -202,11 +202,8 @@ def _getEvaluationContext():
 
 
 def _convertVersionToTuple(version_str):
-    def numberize(v):
-        # For now, we ignore rc/post stuff, hoping it doesn't matter for us.
-        return int("".join(d for d in v if d.isdigit()))
-
-    return tuple(numberize(d) for d in version_str.split("."))
+    # Directly convert the digits in each segment to an integer
```

Comment:
The comment being dropped is not good. Putting `filter` to use accelerates, but lets go of all hope to ever solve comments issue, that 'rc' and so on, may be necessary to be considered ot made visible, the new code makes it harder to see what is happening I feel. 

This code is also not performance relevant at all, so inling a function and thus removing the intent of the code, is removing another kind of comment.